### DISCUSSION TITLE
Ableton origin project file locked by audiogridder

### DISCUSSION THREAD

**Discussion by zbynekdrlik** [apohl79/audiogridder#1366] [Help]
Hi, I am dealing with locked Ableton project file. Save is not possible in first opened project name FOH.als. I always need to save as FOH_A,als and after shutting down ableton need to go and move FOH_A.als to FOH.als.
Ready to provide all asistance and detailed log files but need to be sure that this is live discussion forum. 
Possible same issue here https://github.com/apohl79/audiogridder/issues/1291

**Comment by brkn1** [comment]
same problem here, can't overwrite a project and also the "undo" ableton folder seems to be locked when audiogridder is in use.
